kahuna/public/stylesheets/main.css

```diff
@@ -974,7 +974,8 @@ input.related-labels__parent {
 }
 
 .related-labels__labels {
-    overflow: auto;
+    overflow-x: auto;
```

Comment:
This won't work due to the restrictions behind `overflow`. TL;DR; Anything with `overflow: auto` will have the other axis set to auto too.